config/cc/gcc.in.2

```diff
@@ -37,6 +37,15 @@ config CC_GCC_EXTRA_CONFIG_ARRAY
       if they are properly quoted (or escaped, but prefer quotes). Eg.:
           --with-foo="1st arg with 4 spaces" --with-bar=2nd-arg-without-space
 
+config CC_GCC_MULTILIB_LIST
+    string
+    prompt "List of multilib variants"
+    depends on MULTILIB
+    help
+      Architecture-specific option of expanding or restricting the list of
+      the multilib variants to be built. Refer to GCC installation manual
+      for the format of this option for a particular architecture.
+
```

Comment:
What if the list is empty? Waht about adding:

    Leave empty to use the default list for this architecture.

... or something along those lines...